pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -127,6 +127,7 @@ def _set_device_data(self, device, data):
         data.parents = [d.device_id for d in device.parents]
         data.children = [d.device_id for d in device.children]
         data.is_disk = device.is_disk
+        data.isLeaf = device.isLeaf
```

Comment:
Is it maybe `data.isLeaf = device.is_leaf`?

Reply:
Can you add also maybe a test case at `tests/unit_tests/pyanaconda_tests/modules/storage/test_module_device_tree.py` ?